### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by fsb4000**
English is not my native language. I found these typos using codespell. ( https://pypi.python.org/pypi/codespell/1.9.2 )

### CODE REVIEW

File: doc/release-notes/release-notes-0.4.3.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ http://gitorious.org/bitcoin/bitcoind-stable/archive-tarball/v0.4.3#.tar.gz
 
 BUG FIXES
 
-Cease locking memory used by non-sensitive information (this caused a huge performance hit on some platforms, especially noticable during initial blockchain download).
+Cease locking memory used by non-sensitive information (this caused a huge performance hit on some platforms, especially noticeable during initial blockchain download).
```
Comment (fanquake) [inline doc/release-notes/release-notes-0.4.3.md]: Please don't modify any historical release notes.

**Comment by fsb4000** [comment]
> Please don't modify any historical release notes.

Ok.

**Comment by instagibbs** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/9233/commits/15fa95d7e576041a20d47907356a2be5a327b4ca

**Comment by fanquake** [comment]
utACK 15fa95d

**Comment by kallewoof** [comment]
utACK 15fa95d

**Comment by paveljanik** [comment]
ACK 15fa95d

**Comment by jonasschnelli** [comment]
@fsb4000: can you open a PR upstream for https://github.com/bitcoin/bitcoin/pull/9233/files#diff-dbd6346267524a3c8e3c1c666d7eeaa7?  This change should have been patched upstream in the first place: https://github.com/bitcoin-core/ctaes/blob/master/test.c

File: src/crypto/ctaes/test.c [hunk 2]
```diff
@@ -102,7 +102,7 @@ int main(void) {
         }
     }
     if (fail == 0) {
-        fprintf(stderr, "All tests succesful\n");
+        fprintf(stderr, "All tests successful\n");
```
Comment (MarcoFalke) [inline src/crypto/ctaes/test.c]: @fsb4000 Please submit this upstream, as mentioned by @jonasschnelli .

**Comment by fsb4000** [comment]
@MarcoFalke 
@jonasschnelli 
Sure. Done: https://github.com/bitcoin-core/ctaes/pull/5

**Pull request merged**